images/linux/scripts/base/apt-retry.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+i=0
+tput sc
+echo JustForTestingPurposes
+while fuser /var/lib/dpkg/lock >/dev/null 2>&1 ; do
+    case $(($i % 4)) in
+        0 ) j="-" ;;
+        1 ) j="\\" ;;
+        2 ) j="|" ;;
+        3 ) j="/" ;;
+    esac
+    if [ -n "$TERM" ] && [ "$TERM" = "unknown" ] ; then
+        TERM=rc
+    fi
+    tput rc
+    echo -en "\r[$j] Waiting for other software managers to finish..." 
```

Comment:
Why do we need this logging?

Reply:
Just for troubleshooting purposes

Reply:
I don't think that we should bring such logging to main branch. It looks redundant and unclear. If you prefer, you can leave echo line everything else should be removed